src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] list = new Lottery[3];
+        list[0] = new Lottery();
+        list[1] = new Lottery();
+        list[2] = new Lottery();
```

Comment:
Instead of creating multiple instances of `Lottery` class, you should create only one instance and use it. This will save resources and make the code cleaner.